src/gui/qmlutils.h

```diff
@@ -40,6 +41,8 @@ class OCQuickWidget : public QQuickWidget
     Q_OBJECT
 public:
     using QQuickWidget::QQuickWidget;
+    void setOCContext(const QUrl &src, QWidget *parentWidget, QObject *ocContext, QJSEngine::ObjectOwnership ownership);
```

Comment:
Please tell me what you think, I'm horrible in naming things and not 100% happy with the signature.